pkg/flagparams/flagparams.go

```diff
@@ -27,16 +28,19 @@ func New(filename string, param *v1.ParamSpec) *Flags {
 	stringValues := map[string]*string{}
 	bools := map[string]*bool{}
 	complexValues := map[string]*string{}
+	lists := map[string]*[]string{}
 
 	for _, param := range param.Params {
 		name := strings.ReplaceAll(convert.ToYAMLKey(param.Name), "_", "-")
 		paramToFlag[param.Name] = name
-		if isType(param.Schema, "int") || isType(param.Schema, "uint") {
```

Comment:
I dont know this section of code well. What is Schema vs Type?

Reply:
Changed because @ibuildthecloud introduced the new `Type` field for the params and asked me to use it here.
Schema can be e.g. `*false | bool` while type is just the type, e.g. `bool`